test/snippet/range/view/take_line_adaptor_def.cpp

```diff
@@ -0,0 +1,13 @@
+#include <string>
+
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/std/ranges>
+
+int main()
+{
+    std::view::take_while([] (auto const & l) { return (l != '\r') && (l != '\n'); });
```

Comment:
What does this line do? And why does it appear with `view::take_line`?

Reply:
I'm not sure, it was already a snippet before. You wrote the file, I think?

I guess you were trying to show that `view::take_line` is equivalent to the line above, except for single pass input ranges?

Reply:
I can just delete the snippet and change this part:
```
* This adaptor returns a single line **excluding** the end-line character(s), *but moving the cursor behind them
 * for single-pass ranges.* I.e. for all ranges that satisfy std::ranges::ForwardRange this is the same as calling
 * ranges::view::take_while:
 * \include test/snippet/range/view/take_line_adaptor_def.cpp
 * but for *single pass input ranges* this means that any endline characters after the returned range are also consumed
 * (this potentially includes multiple newline characters).
```
to:
```
* This adaptor returns a single line **excluding** the end-line character(s), *but moving the cursor behind them
 * for single-pass ranges.* I.e. for all ranges that satisfy std::ranges::ForwardRange this is the same as calling
 * ranges::view::take_while, but for *single pass input ranges* this means that any endline characters after the returned range are also consumed
 * (this potentially includes multiple newline characters).
```

Reply:
Hm, I think it's one of the places where there shouldn't be a copy'n'pastable snippet, because it's not supposed to be functioning program. It's just "this means that". I would leave the original one-liner inside the documentation and just not make a snippet. Or make a snippet, but than include it the old way via`\snippet`.